go/src/koding/kites/kloud/provider/aws/aws.go

```diff
@@ -202,10 +202,10 @@ func (meta *Cred) Valid() error {
 		return errors.New("aws: secret key is empty")
 	}
 	if !reAccessKey.MatchString(meta.AccessKey) {
-		return errors.New("aws: access key is ill-formed")
+		return errors.New("aws: access key seems wrong")
```

Comment:
> i thought that was the intention (we don't know if it's wrong or not)

Actually, If this sentence was `access key format seems wrong`, It would be correct, since Amazon can change their key formats anytime.